app/forms/recovery_code_form.rb

```diff
@@ -25,9 +25,9 @@ def valid_recovery_code?
     recovery_code_generator.verify(code)
   end
 
-  def result
+  def extra_analytics_attributes
     {
-      success: success,
+      method: 'recovery code',
```

Comment:
I think renaming the hash key, we you suggested, is a great idea and clarifies this without the need for a constant! 🌈